src/include/cpp-btree/btree.h

```diff
@@ -704,21 +653,21 @@ class btree_node {
 
   // Removes the value at position i, shifting all existing values and children
   // at positions > i to the left by 1.
-  void remove_value(int i, allocator_type *alloc);
+  void remove_value(const int i, allocator_type *alloc);
 
   // Removes the values at positions [i, i + to_erase), shifting all values
   // after that range to the left by to_erase. Does not change children at all.
   void remove_values_ignore_children(int i, int to_erase,
                                      allocator_type *alloc);
 
   // Rebalances a node with its right sibling.
-  void rebalance_right_to_left(int to_move, btree_node *right,
+  void rebalance_right_to_left(const int to_move, btree_node *right,
```

Comment:
I'm not sure we need to reflect the constness at the declaration level.

Reply:
i just want to make it consistent.

Reply:
OK, I'm fine with that.